docs/okta/native-apps-integration-guide.md

```diff
@@ -239,4 +239,44 @@ note over NativeLayer: the SDK manages the tokens, which can now be used to<br/>
 
 ## Fallback
 
-TODO
+While we are confident that the above approach is robust enough to work, we will also include a fallback option in case something does go wrong with the above approach.
+
+Specifically a fallback option will be used if we're unable to intercept the `fromURI` parameter from Okta.
+
+To handle this fallback option, a [private-use URI scheme](https://datatracker.ietf.org/doc/html/rfc8252#section-7.1) (referred to as "custom URL scheme") will have to be registered with the app. We suggest a custom URL scheme for this URI, which we can identify (`your-app://identity/fallback`) e.g. `com.theguardian.app://identity/fallback`.
```

Comment:
is it worth mentioning that the custom url also need to be added as a valid return url in the app okta config too ?

Reply:
Good idea, I'll add that in!